### DISCUSSION TITLE
How do I do step 3. of installation: "Set your vault path in the workflow configuration."?

### DISCUSSION THREAD

**Discussion by ccropper** [chrisgrieser/shimmering-obsidian#174] [Q&A] [answered]
I'm fairly new to Alfred and Obsidian, and I could not figure out how to do this step:

3. Set your vault path in the workflow configuration.

I'm on MacOS. Where do I go to add my vault path?

**Comment by chrisgrieser** [comment, marked as answer]
It's referring to the configuration inside the Alfred Preferences
![Pasted image 2024-04-26 at 16 17 41@2x](https://github.com/chrisgrieser/shimmering-obsidian/assets/73286100/bff352ac-ce3e-4302-b17d-bc64dedb7416)

> I'm fairly new to Alfred and Obsidian

Just an advice: I'd maybe hold off on using Shimmering Obsidian for a bit? Both, Alfred and Obsidian are apps that have a bit of a learning curve to understand all the features. Learning both at the same time, and also one of learning Shimmering Obsidian on top of that, might be a bit much at once.

**Reply by ccropper** [reply]
Ah, thank you. This was definitely user error as I was still on Alfred 4! You clearly stated 

> This workflow requires Alfred 5 with [Powerpack](https://www.alfredapp.com/powerpack/).

Embarrassing 🙈
